Cargo.toml

```diff
@@ -10,24 +10,35 @@ name = "cursive"
 readme = "Readme.md"
 repository = "https://github.com/gyscos/Cursive"
 version = "0.3.1"
+build = "build.rs"
 
 [build-dependencies]
 skeptic = "0.6"
 
+[features]
+cursive_ncurses = ["ncurses"]
```

Comment:
The way it was before was working well. You don't need to _define_ the ncurses, pancurses or termion features, but you can still refer to them in the `default` array or in the `#[cfg]` directives.